Update programmatically-hidden.md (#1764)
act-rules/act-rules.github.io: pages/glossary/programmatically-hidden.md

An HTML element is _programmatically hidden_ if either it has a [computed][] CSS property `visibility` whose value is not `visible`; or at least one of the following is true for any of its [inclusive ancestors][] in the [flat tree][]:

- has a [computed][] CSS property `display` of `none`; or
- has an `aria-hidden` attribute set to `true`

**Note**: Contrarily to the other conditions, the `visibility` CSS property may be reverted by descendants.

> **Note**: The [HTML standard suggests](https://html.spec.whatwg.org/multipage/rendering.html#hidden-elements) rendering elements with the `hidden` attribute with a CSS rule that applies the value `none` to the CSS property `display` of the element. Although the suggestion is not normative, know user agents render it according to the suggestion. If a user agent does not follow the suggestion, this definition will not apply in the user agent's context.

Reviewer: ```suggestion
**Note**: The [HTML standard suggests](https://html.spec.whatwg.org/multipage/rendering.html#hidden-elements) rendering elements with the `hidden` attribute with a CSS rule that applies the value `none` to the CSS property `display` of the element. Although the suggestion is not normative, know user agents render it according to the suggestion. If a user agent does not follow the suggestion, this definition may produce incorrect results for this user agent.
```
making it a bit less dramatic.

> **Note**: The [HTML standard suggests](https://html.spec.whatwg.org/multipage/rendering.html#hidden-elements) rendering elements with the `hidden` attribute with a CSS rule that applies the value `none` to the CSS property `display` of the element. Although the suggestion is not normative, know user agents render it according to the suggestion. If a user agent does not follow the suggestion, this definition will not apply in the user agent's context.

Reviewer 2: I would like to know what AT does with `hidden style="display:block"`. I think we should test this scenario, just to be safe.

Author: A `style` attribute setting the value of `display` overrules the `hidden` attribute. Irrespective of the CSS being declared as external, internal or inline. I've clarified that in the note.
